Description: Remove dependency cradle, seems unused
Bug: https://github.com/casey/just/issues/1679
Last-Update: 2023-09-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -141,9 +141,6 @@
 version = "1.0.0"
 features = ["v4"]
 
-[dev-dependencies.cradle]
-version = "0.2.0"
-
 [dev-dependencies.executable-path]
 version = "1.0.0"
 
--- a/tests/lib.rs
+++ b/tests/lib.rs
@@ -5,7 +5,6 @@
     tempdir::tempdir,
     test::{Output, Test},
   },
-  cradle::input::Input,
   executable_path::executable_path,
   just::unindent,
   libc::{EXIT_FAILURE, EXIT_SUCCESS},
